demos/device_defender_for_aws/metrics_collector/lwip/metrics_collector.c

```diff
@@ -0,0 +1,254 @@
+/* Standard includes. */
```

Comment:
Can you add the license in this file.

Reply:
Done by 3238843045b336857ba459da6f72415ed5a14400

Reply:
Done by commit 3238843